cluster-provision/k8s/1.24/provision.sh

```diff
@@ -76,6 +76,10 @@ echo 'ACTION=="add|change", SUBSYSTEM=="block", KERNEL=="vd[a-z]", ATTR{queue/ro
 # To prevent preflight issue related to tc not found
 dnf install -y tc
 
+# The selinux-policy package shipped with the latest Centos stream "release" can be outdated.
+# Example: Centos 8 20220913.0 ships with selinux-policy-3.14.3-108, which misses crucial permissions
+dnf -y update selinux-policy
```

Comment:
can we add a check that if the permissions exists fail before we update ?
so once it is fixed we can drop this, but not a must if we will remember / open issue about it that will remind us

Reply:
> can we add a check that if the permissions exists fail before we update ? so once it is fixed we can drop this, but not a must if we will remember / open issue about it that will remind us

That requires installing `setools`, which comes with 129 dependent packages...
TBH I feel like it kinda makes sense to run with the latest SELinux policy always, but I'm fine opening an issue if that's not desirable.

> 
> Beside that lgtm Thanks was it tested with kubevirt jobs manually ?

I did make sure that `dnf -y update selinux-policy` updates to a version that brings back the permissions we care about, but I did not run `make functest` on it.

Reply:
Imho having an issue is better please, bundle is preferable and easier to maintain

about `functest` would you please do ssh.sh to the VMs on the current provider, to install it as a patch and run for example this lane `pull-kubevirt-e2e-k8s-1.24-sig-network` ?
unless you know for sure this is the problem and we that we can go blindly

Reply:
Issue is fine thanks

About `DisableCustomSELinuxPolicy` do we need to change kubevirt then ?

Reply:
/cc @AlonaKaplan @phoracek

Reply:
We can try to move all the passt tests to an unpriveleged namespace (and disable the custom selinux policy). I believe it should work.

Reply:
The passt tests should actually run fine with the latest Centos 8 under container_t. In fact, Centos 8 now seems to behave mostly like Centos 9, where loading a custom policy fails...

Reply:
we dont need to do anything with kubevirt about `DisableCustomSELinuxPolicy` ?
Passt should pass out of the box ?

Reply:
We most likely do need to set that feature gate on all lanes.
Waiting to see what happens in https://github.com/kubevirt/kubevirt/pull/9208, with https://github.com/kubevirt/kubevirt/pull/9373 standing by.